layouts/index.html

```diff
@@ -54,9 +60,14 @@ <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
         var button = document.getElementById( 'js-user-submit' );
         button.addEventListener( 'click', function ( event ) {
           event.preventDefault();
+          {{ if eq $.Site.Params.language "english" }}
+          var registerPath = "register/";
```

Comment:
Yeah, I'm not sure how to tackle this better, but I do know that for now this JS is very barebones and just running on the homepage. I'll open an issue around this.

```
       // TODO: This exists only for basic testing. It will be removed once
       // the designs are more fleshed out. This is just a work in progress for now.
```